src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,30 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { user, comments, body, title } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo user={user} />
```

Comment:
user can be null, so let's render this component only when the user exists
```suggestion
          {user && <UserInfo user={user} />}
```